docs/Guides/Ecosystem.md

```diff
@@ -635,7 +635,10 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+- [`@clerk/fastify`](https://github.com/clerkinc/javascript/tree/main/packages/fastify) Clerk is the easiest way to add authentication and user management to your Fastify application.
```

Comment:
Can we please not have advertisement showed down our throat? Is Clerk really the easiest way to add authentication and user management to fastify?

Reply:
I just reused the existing Readme from the clerk repo but I'll be more than happy to change accordingly 😅 .